src/Driver/PhpAmqpLib/Queue.php

```diff
@@ -304,7 +304,7 @@ public function cancel(string $consumerTag = '')
         $this->channel->getResource()->basic_cancel(
             $consumerTag,
             (bool) ($this->flags & Constants::AMQP_NOWAIT),
-            false
+            true
```

Comment:
why is this change needed? you can control this behaviour this the AMQP_NOWAIT flag.

Reply:
ping @oqq

Reply:
But is AMQP_NOWAIT always set?

How I have tested this, basic_cancel must called with ($tag, false, true) if the server should be stopped. Maybe we add another method "stop" here and hard set (false, true)?